index.js

```diff
@@ -18,7 +18,6 @@ var resolveMedia = require("./lib/resolve-media")
 var moduleDirectories = [
   "web_modules",
   "node_modules",
-  "bower_components",
```

Comment:
This should be added in the CHANGELOG since it's a breaking changes (and we should add how to add bower support like it was).